packages/react-instantsearch-dom/src/components/HierarchicalMenu.js

```diff
@@ -28,6 +27,7 @@ class HierarchicalMenu extends Component {
     className: PropTypes.string,
     limit: PropTypes.number,
     showMoreLimit: PropTypes.number,
+    isEmpty: PropTypes.bool,
```

Comment:
Does this exist? I can't find it from [the doc](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/react/) and `connectHierarchicalMenu.js`.

Reply:
yes, I guess we should remove it here and in other parts of the code that assume there is a isEmpty prop.

Reply:
there's no more instances of the isEmpty prop, so this removal removed them all